src/shared/components/search-filters/search-filters.component.html

```diff
@@ -17,11 +17,13 @@
     </select>
   </div>
 
-  <div id="dynamicFilters" *ngIf="isSearchableAndSearchInputsReady()">
-    <div class="form-group" *ngFor="let searchInput of searchInputs">
-      <ccd-field-write [caseField]="searchInput.field" [formGroup]="formGroup" [isExpanded]="true" (keyup.enter)="apply()"></ccd-field-write>
+  <ng-container ccdConditionalShowForm *ngIf="isSearchableAndSearchInputsReady()" [formGroup]="formGroup" [contextFields]="caseFields">
+    <div id="dynamicFilters">
+      <div class="form-group" *ngFor="let searchInput of searchInputs">
```

Comment:
_Question:_
What happens to this `<div>` when a field is hidden? Does it collapse away entirely? 🤔

Reply:
Nope - if there is a jurisdiction and a case type and the getSearchInputs method has finished doing it's thing it will be shown.

Reply:
But the `<ccd-field-write></ccd-field-write>`, below, will hide or show according to the logic on the `show_condition` for the field, right? I'm wondering what happens to this wrapper `<div>`... because of the `show_condition` is not met, you'll have an empty `<div>` and I'm wondering if it takes up spaces at all.

Reply:
The wrapper div (dynamicFilters) has always been present on the page irrespective of whether or not there are any search inputs. There are lots of tests searching for the wrapper div so I decided not to add conditional logic to hide it if there are no searchInputs.